service-base/src/main/java/org/eclipse/hono/service/AbstractServiceBase.java

```diff
@@ -432,6 +432,9 @@ protected final void addTlsKeyCertOptions(final NetServerOptions serverOptions)
                 log.info("enabling secure protocol [{}]", protocol);
                 serverOptions.addEnabledSecureTransportProtocol(protocol);
             });
+
+            serverOptions.setSni(getConfig().isSni());
+            log.info("enabling TLS SNI [{}]", getConfig().isSni());
```

Comment:
how about logging this line *only* if we are actually enabling SNI? Otherwise, maybe change the log statement to
> Service supports TLS ServerNameIndication: {}